zebra-network/src/config.rs

```diff
@@ -34,6 +34,10 @@ pub struct Config {
     pub peerset_initial_target_size: usize,
 
     /// How frequently we attempt to connect to a new peer.
+    ///
+    /// This duration only pertains to the rate at which zebra crawls for new
+    /// peers, not the rate zebra connects to new peers, which is restricted to
+    /// CandidateSet::PEER_CONNECTION_INTERVAL (100 milliseconds)
     pub new_peer_interval: Duration,
```

Comment:
Hmm maybe we should also do a rename here, what do you think of `peer_gossip_request_interval`?

Reply:
Also, I prefer not to copy constant values into comments, because they get out of sync with the code. This is particularly important for security parameters.